pkg/minishift/network/settings_linux.go

```diff
@@ -23,3 +23,7 @@ import (
 func ConfigureNetworking(machineName string, networkSettings NetworkSettings) {
 	fmt.Println(configureIPAddressMessage, configureIPAddressFailure)
 }
+
+func determineNameservers() string {
+	return "" // not implemented
+}
```

Comment:
Does that mean it won't work with Virtualbox on top of Linux?

Reply:
Determining nameservers is only used in the hyperv path of code. Implementation is needed to compile, but none is needed, as this only works on Windows